asm/custom-functions/src/lib.rs

```diff
@@ -404,6 +433,33 @@ fn randomize_boss_key_start_pos(ptr: *mut u16, mut seed: u32) {
     }
 }
 
+#[no_mangle]
+fn enforce_loftwing_speed_cap(loftwing_ptr: *mut AcOBird) {
+    let loftwing = unsafe { &mut *loftwing_ptr };
+    let mut is_in_levias_fight = false;
+    if unsafe { &SPAWN_SLAVE.name[..4] } == b"F023" {
+        let levias_ptr = unsafe {
+            findActorByActorType(184 /* NusiB */, ptr::null())
+        };
+        if !levias_ptr.is_null() {
+            if unsafe { checkXZDistanceFromLink(levias_ptr, 5000f32) } {
```

Comment:
```suggestion
            if unsafe { checkXZDistanceFromLink(levias_ptr, 20000f32) } {
```
The range is a bit small atm. Tried it with 20000 and it slowed down much further away - enough that I didn't accidentally crash into Levias.